torch/csrc/distributed/rpc/rref_context.cpp

```diff
@@ -385,21 +389,27 @@ std::shared_ptr<Future<c10::intrusive_ptr<OwnerRRef>>> RRefContext::
     // Scenario (1) RRef is used before it is created
     const auto pendingOwnerIter = pendingOwners_.find(rrefId);
     if (pendingOwnerIter == pendingOwners_.end()) {
+      // Note: The type passed into RRefType::create() does not matter here, as
+      // the future is marked as completed with the RRef of the specific type
+      // in getOrCreateOwnerRRef().
```

Comment:
is there any concern if RRefType's type does not match with the real RRef value?

cc @wanchaol

Reply:
I didn't see any failures with this presumably since we are filling it with the right type (more specific than `Any` which is the most broad) when the owner RRef is created. Although, would be good for @wanchaol to clarify this.

Reply:
it should be fine as we fill the type later when we actually create the rref, here is merely a placeholder for this rref_id i think.